models/user.go

```diff
@@ -119,6 +121,10 @@ func (u *User) BeforeUpdate() {
 	u.UpdatedUnix = time.Now().Unix()
 }
 
+func (u *User) AfterLogin() {
```

Comment:
@0xbaadf00d I chose this name because it is generic, it may be that in the future another implementation can be done in this same method. But I can change if is necessary.